rj_gameplay/stp/role/assignment/naive.py

```diff
@@ -109,11 +112,11 @@ def compute_costs_matrix(
 
             # Throw an exception if the returned cost is not finite.
             if not isfinite(unassigned_cost):
-                    raise ValueError(
-                        "Got a non-finite cost ({}) for request {} and unassinged robot".format(
-                            cost, request, robot
-                        )
+                raise ValueError(
+                    "Got a non-finite cost ({}) for request {} and unassinged robot".format(
```

Comment:
```suggestion
                    "Got a non-finite cost ({}) for request {} and unassigned robot {}".format(
```

will fix the style complain about "unused arguments"